package.json

```diff
@@ -42,8 +42,10 @@
     "validate-peer-dependencies": "^1.2.0"
   },
   "devDependencies": {
+    "@ember/optional-features": "^2.0.0",
     "@ember/test-helpers": "^2.4.2",
     "babel-eslint": "^10.1.0",
+    "ember-angle-bracket-invocation-polyfill": "^3.0.1",
```

Comment:
To make sure `<LinkTo @route="posts">posts</LinkTo>` also works in the `ember-lts-3.8` scenario.